MSAL/src/public/MSALDefinitions.h

```diff
@@ -117,5 +117,11 @@ typedef void (^MSALAccountsCompletionBlock)(NSArray<MSALAccount *> * _Nullable a
  */
 typedef void (^MSALLogCallback)(MSALLogLevel level, NSString * _Nullable message, BOOL containsPII);
 
+/*!
+ MSAL telemetry callback.
+ 
+ @param events Array of Dictionaries keyed on event name, valued on event data.
```

Comment:
If we only do aggregated data now, do we need to return an array of dictionaries, or a single dictionary should be sufficient?

Reply:
Changed to singe dictionary.